### PR TITLE
Removing IWR and adding curl

### PR DESCRIPTION
**Pull request by RAhnemann**
Removing the Invoke-WebRequest for standardization.  Also leveraged existing format of the C:\downloads folder for downloads and expanding into appropriate destination.

### CODE REVIEW

File: windows/9.0.x/sitecore-xm-solr/Dockerfile [hunk 1]
```diff
@@ -9,9 +9,9 @@ SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPref
 ARG SOLR_VERSION
 ARG CORE_NAMES
 
-RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; `
-    Invoke-WebRequest -Uri ('https://archive.apache.org/dist/lucene/solr/{0}/solr-{0}.zip' -f $env:SOLR_VERSION) -OutFile 'C:\\solr.zip'; `
-    Expand-Archive -Path 'C:\\solr.zip' -DestinationPath 'C:\\temp'; `
+RUN New-Item -Path 'C:\\downloads' -ItemType 'Directory' -Force | Out-Null; `
```
Comment (jeanfrancoislarente) [inline windows/9.0.x/sitecore-xm-solr/Dockerfile]: Is the TLS entry still not an issue? I thought that on some machines this entry was required to ensure the correct version of Tls was used.

File: windows/dependencies/mssql-developer-2017/Dockerfile [hunk 2]
```diff
@@ -12,15 +12,15 @@ ENV sa_password='_' `
 
 RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; `
```
Comment (pbering) [inline windows/dependencies/mssql-developer-2017/Dockerfile]: ServicePointManager hack should also be removed here

File: windows/9.0.x/sitecore-xm-solr/Dockerfile (hunk 1 above)
Comment (pbering) [reply windows/9.0.x/sitecore-xm-solr/Dockerfile]: ServicePointManager is only for .NET / PowerShell, curl.exe is using "something" else.

File: windows/dependencies/mssql-developer-2017/Dockerfile (hunk 2 above)
Comment (pbering) [inline windows/dependencies/mssql-developer-2017/Dockerfile]: ServicePointManager usage should also be removed here

File: windows/9.0.x/sitecore-xm-solr/Dockerfile (hunk 1 above)
Comment (RAhnemann) [reply windows/9.0.x/sitecore-xm-solr/Dockerfile]: By default, curl.exe uses the highest level TLS available [https://ec.haxx.se/usingcurl-tls.html](https://ec.haxx.se/usingcurl-tls.html)

**Pull request merged**